CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Mattermost Mobile Apps Changelog
 
+## 1.26.1 Release
+- Release Date: 
+- Server Versions Supported: Server v5.9+ is required, Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device
+
+### Compatibility
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).
+ - iPhone 5s devices and later with iOS 11+ is required.
+
+### Bug Fixes
+ - 
+
```

Comment:
* Fixes a crash on Android and iOS on servers prior to the ESR (not sure if we should add this)
* Fixes a crash when connecting the WebSocket to a server with CBA enabled